workspace/viewer/PdfJsViewer.ts

```diff
@@ -440,8 +440,8 @@ export class PdfJsViewer {
       let scale = this.zoom.factor(page, this.jqRoot);
       let pageView;
       if (this.currentTask.isResize) {
-        this.resetPage();
```

Comment:
Why did you remove this? It means that  you no longer call `zoom.onResize` which is probably doing some useful stuff (and it really does: it resets scale so that it was calculated the next time `factor` is called, so that page size could beset properly after viewer container resizing)

Reply:
I removed it because `setFitting()` function does exactly the same: changes `fittingScale` to `undefined` to recalculate size next time in cases of `ZoomingMode.FIT_PAGE`, `ZoomingMode.FIT_WIDTH`. Case of `ZoomingMode.PRESET` doesn't need `onResize` function to be called.

Reply:
Why don't you call that function then? Before this change when flag `isResize` was set in the task it would resize page appropriately but now when this code is removed it won't.

Reply:
Function `setFitting()` is called, but in `ZoomWidth()`, `ZoomPreset()`, `ZoomPage()`, which are executed before `openPage()` function, so it works fine.
I dont understand why we should evoke `resetPage()` if `setFitting()` does exactly the same?

Reply:
Anyway, I returned function `resetPage()` back to keep going on.

Reply:
Thanks for returning it back.

It is not the question of calling `resetPage` vs calling `setFitting`. It is the question of calling _anything_ which resets zooming level when we find a task with `resize == true`. Before these changes it was called and code in Papeeria which adds resize tasks expected such behavior. After these changes we stopped calling, and code in Papeeria got broken (for the user it meant that page won't resize properly when he resizes the viewer pane). 

But anyway we don't even  add such tasks on resize now.
Please replace onResize impementation with this one:
```
  onResize() {
    if (this.currentFile && this.currentFileUrl) {
      this.showAll(this.currentFileUrl, true);
    }
  }
```

Reply:
It's clear. Thanks for explanation.
Done.